vanilla/vanilla
# Remove old redux utilities and refactor auth pages

Closes https://github.com/vanilla/vanilla/issues/7952

- Deletes `@libary/redux/utility`
- Refactors the only components using them, the shelved, feature-flagged, auth pages.
  - Now uses functional components.
  - Now use hooks.
  - Now has a single unified reducer.
  - Now uses the new action class, and FSA actions.
  - Preloads authenticator data properly.
  - Fixed error handling which was not functioning correctly.
- KB has 1 import to update as well, that can be found here https://github.com/vanilla/knowledge/pull/1452


## Testing

To see the auth pages:

1. Add the following line to your config.

```php
$Configuration['Feature']['AuthenticationAPI']['Enabled'] = true;
```

2. Go to `/authenticate` on your local site.
3. Login & password reset are the only implemented features.
